# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/prim_store/picture.rs
# Commit: beec388d4b0e
# Full Hash: beec388d4b0e6a50ac243ed86c0e3f1f48cbe744
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-06-25 09:36:02
# Regressor Bug: 1896503
# File Overlap Count: 1
# Description:
#   Bug 1896503 - Implement FilterInstance code to send SVG filter graph to WebRender r=mstange,gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D174208
# ==============================================================================

diff -r 2147e0051e23 -r beec388d4b0e gfx/wr/webrender/src/prim_store/picture.rs
--- a/gfx/wr/webrender/src/prim_store/picture.rs	Mon Jun 24 23:40:38 2024 +0000
+++ b/gfx/wr/webrender/src/prim_store/picture.rs	Mon Jun 24 23:45:05 2024 +0000
@@ -795,9 +795,6 @@
     /// sRGB texture pixel colors on load and convert back on store, for correct
     /// interpolation
     pub linear: bool,
-    /// padding for output rect if we need a border to get correct clamping, or
-    /// to account for larger final subregion than source rect (see bug 1869672)
-    pub inflate: i16,
     /// virtualized picture input binding 1 (i.e. texture source), typically
     /// this is used, but certain filters do not use it
     pub inputs: Vec<FilterGraphPictureReferenceKey>,
@@ -811,7 +808,6 @@
         FilterGraphNodeKey{
             kept_by_optimizer: node.kept_by_optimizer,
             linear: node.linear,
-            inflate: node.inflate,
             inputs: node.inputs.into_iter().map(|node| {node.into()}).collect(),
             subregion: [
                 Au::from_f32_px(node.subregion.min.x),